catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -221,13 +230,16 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+
                         <Embed-Dependency>
                             catalog-core-solr,
                             guava,
```

Comment:
Yup. Was going to pull it out after I finish. Didn't want to muck with it while I was getting stuff to work.